vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -426,14 +440,29 @@ def grouped_topk(hidden_states: torch.Tensor,
     return topk_weights, topk_ids
 
 
+def get_config_dtype_str(dtype: torch.dtype,
+                         use_int8_w8a16: Optional[bool] = False,
+                         use_fp8_w8a8: Optional[bool] = False):
+    if use_fp8_w8a8:
+        return "fp8"
```

Comment:
```suggestion
        return "fp8_w8a8"
```

Reply:
I'll change the moe config filename as well.